src/style.css

```diff
@@ -1 +1,60 @@
-/* Styles go here */
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin-top: 22px;
+  margin-bottom: 23px;
+}
+
+.nav__item {
+  height: 15px;
+  margin-left: 20px;
```

Comment:
Here you should use `:not(:last-child)` not to allow this child goes through its parent